server/models/article.js

```diff
@@ -38,6 +57,42 @@ articleSchema.statics.urisAreEqual = (uri1, uri2) => {
   return normalizeURI(uri1) === normalizeURI(uri2);
 };
 
+articleSchema.methods.getMercuryInfo = function getMercuryInfo() {
```

Comment:
so this method is essentially a fetch to grab data on an article? 

would maybe rename to "fetchMercuryInfo" because "get" sounds like it's just returning the "info" field on an article, not actually populating the info field